Task1/src/task3/task1_ES6.js

```diff
@@ -0,0 +1,12 @@
+import readline from 'readline';
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    terminal: false
+});
+
+rl.on('line', input => {
+    const reversedInput = [...input].reverse().join('');
+    process.stdout.write(reversedInput);
```

Comment:
It would be nice for a better user experience to move the caret to the new line after result is shown.